Advanced_Tutorials/Object Detection with CURE-OR.ipynb

```diff
@@ -0,0 +1,1404 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Objective\n",
+    "The objective of this exercise is to familiarize you with the \"Challenging Unreal and Real Environments for Object Recognition (CURE-OR)\" Dataset. This is one of the largest datasets generated as of yet to help benchmark studies regarding the robustness of object recognition algorithms under a variety of different conditions. For our purposes, we will be using a smaller version of this dataset, called \"mini CURE-OR\", that contains a smaller subset of the larger version for a variety of different backgrounds, camera poses, noise etc."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Downloading Files\n",
+    "\n",
+    "We download datafiles using the links provided below. It is highly recommended that you do not change them. You will have to do this only once."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "100% [............................................................................] 153396 / 153396"
+     ]
+    }
+   ],
+   "source": [
+    "import wget\n",
```

Comment:
We might add the command   `!pip install wget` before importing the wget package in case the first time to run the notebook. I installed wget then downloaded the data files.